src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/MavenVerifier.java

```diff
@@ -295,6 +310,59 @@ private void checkSoftwareConfigurationManagementField(Model model, HashSet<Veri
             }
         }
     }
+
+    private void checkDependencyManagement(Model model, HashSet<VerificationMessage> hostingIssues) {
```

Comment:
The dependencyManagement check is not 100% accurate. Ideally we would read the pom of the bom and it's transitive poms and fetch all dependencies from there to see what is managed by the bom. And then we can loop over all the dependencies and check if they are contained in the bom.
Right now it could happen that it complains about a missing dependencyManagement section although there is no dependency that could be managed via the bom.